Library/Formula/sstp-client.rb

```diff
@@ -0,0 +1,33 @@
+class SstpClient < Formula
+  homepage "http://sstp-client.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/sstp-client/sstp-client/1.0.9/sstp-client-1.0.9.tar.gz"
+  sha1 "b56c60fc6ecd8b1686d6c319a8adb3f988bcabbd"
+
+  depends_on "libevent"
+  depends_on "openssl"
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--disable-ppp-plugin",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+
+    # Create a directory needed by sstpc for privilege separation
+    mkdir_p "var/run/sstpc"
```

Comment:
I just realized that this line does not create the required folder properly; it should be `mkdir_p var/"run/sstpc"`. Also, there is one more option that should be passed to `configure` (`--with-runtime-dir=#{var}/run/sstpc`) Shall I close the pull request and start a new one or is it possible to update this somehow?